src/borg/archiver.py

```diff
@@ -2694,6 +2708,20 @@ def define_borg_mount(parser):
           option is internally enforced by borg. "ignore_permissions" can be given to
           not enforce "default_permissions".
 
+        Option ``--bypass-lock`` allows you to access the repository while bypassing
+        borg's locking mechanism. This is necessary if your repository is on a read-only
+        storage where you don't have write permissions or capabilities and therefore
+        cannot acquire a lock. Examples are repositories stored on a Bluray disc or a
+        read-only network storage. Avoid this option if you are able to use locks as
+        it's the safer way; see the warning below.
+
+        .. note::
+
```

Comment:
is there also **warning** instead **note**?

Reply:
Yes, Sphinx/RST has `.. warning::` I only initially used `.. note::` because most other "warnings" are notes, too, like https://github.com/borgbackup/borg/blob/master/docs/usage/config.rst Changed it. :+1: